pkg/oci/mutate/signatures.go

```diff
@@ -36,10 +38,15 @@ func AppendSignatures(base oci.Signatures, sigs ...oci.Signature) (oci.Signature
 			Annotations: ann,
 		})
 	}
+
 	img, err := mutate.Append(base, adds...)
+
+	// Set the Created date to time of execution
+	img, _ = mutate.CreatedAt(img, v1.Time{Time: time.Now()})
```

Comment:
We should check both of these errs. With this, we'll return `mutate.Append`'s err, but only after calling `mutate.CreatedAt`.